hedera-node/src/main/java/com/hedera/services/sigs/metadata/DelegatingSigMetadataLookup.java

```diff
@@ -185,6 +185,11 @@ public DelegatingSigMetadataLookup(
 		return accountSigMetaLookup.safeLookup(id);
 	}
 
+	@Override
```

Comment:
We no longer need this method (or `SigMetadataLookup.pureAccountSigningMetaFor()`) because we want to accept the [`master-plus-sdk-0.15.0` implementation](https://github.com/hashgraph/hedera-services/blob/master-plus-sdk-0.15.0/hedera-node/src/main/java/com/hedera/services/sigs/metadata/lookups/BackedAccountLookup.java) of `BackedAccountsLookup`, which will never result in a call to `getForModify()` during key expansion.

Reply:
As discussed via slack, we will have a separate issue and PR to adapt to the new implementation of platform SDK 0.15.0